packages/surge-xt/PKGBUILD

```
@@ -0,0 +1,104 @@
+  'jack: backend for standalone'
+)
+# checkdepends=('lv2lint')
+source=("${pkgname}-${pkgver}.tgz::https://github.com/surge-synthesizer/releases-xt/releases/download/${pkgver}/surge-src-${pkgver}.tar.gz")
+sha256sums=('100b536f741c62529800fc26e3773567591385d4152db305382b4315f6e39c55')
+
+prepare() {
+  mv -v "$_repo_name" "$pkgname-$pkgver"
+  cd "$pkgname-$pkgver"
+  # first shot at improving reproducibility:
+  # https://github.com/surge-synthesizer/surge/issues/2389
+  sed -e 's/@SURGE_BUILD_DATE@/0/g' \
+      -e 's/@SURGE_BUILD_TIME@/0/g' \
+      -e 's/@SURGE_BUILD_HASH@/0/g' \
+      -e 's/@SURGE_BUILD_FQDN@/None/g' \
+      -e "s/@SURGE_BUILD_ARCH@/${CARCH}/g" \
+      -e "s/@SURGE_FULL_VERSION@/${pkgver}/g" \
+      -i src/common/version.cpp.in
+}
+
+build() {
+  cd "$pkgname-$pkgver"
+  # binaries and shared libs have executable stack:
+  # https://github.com/surge-synthesizer/surge/issues/1701
+  export LDFLAGS="${LDFLAGS},-z,noexecstack"
+  # build surge-headless
+  cmake -DCMAKE_INSTALL_PREFIX='/usr' \
+        -DCMAKE_BUILD_TYPE='None' \
+        -W no-dev \
+        -B build \
+        -S .
+  make VERBOSE=1 -C build
+}
+
+# check() {
+#   cd "$pkgname-$pkgver"
+#   lv2lint -Mpack -I "build-test/surge_xt_products/${_name}.lv2" \
+#     "https://surge-synthesizer.github.io/lv2/surge" \
+#     || echo "Known to fail: https://github.com/surge-synthesizer/surge/issues/2392"
+# 
+#   mkdir -vp test_data/${pkgname}
+#   cp -av resources/data/* test_data/${pkgname}
+#   XDG_DATA_HOME=test_data build/surge-headless
+# }
+
+package() {
+  depends+=('libfreetype.so')
+  cd "$pkgname-$pkgver"
+
+  # install lv2
+#   install -vDm 755 "build/surge_xt_products/${_name}.lv2/${_name}.so" \
+#     -t "${pkgdir}/usr/lib/lv2/${_name}.lv2/"
+#   install -vDm 644 "build/surge_xt_products/${_name}.lv2/"*.ttl \
+#     -t "${pkgdir}/usr/lib/lv2/${_name}.lv2/"
+#   install -vDm 755 "build/surge_xt_products/${_name} Effects.lv2/${_name} Effects.so" \
+#     -t "${pkgdir}/usr/lib/lv2/${_name} Effects.lv2/"
+#   install -vDm 644 "build/surge_xt_products/${_name} Effects.lv2/"*.ttl \
+#     -t "${pkgdir}/usr/lib/lv2/${_name} Effects.lv2/"
+
+  # install vst3
+  install -vDm 755 "build/surge_xt_products/${_name}.vst3/Contents/${CARCH}-linux/${_name}.so" \
+    -t "${pkgdir}/usr/lib/vst3/${_name}.vst3/Contents/${CARCH}-linux/"
+  install -vDm 755 "build/surge_xt_products/${_name} Effects.vst3/Contents/${CARCH}-linux/${_name} Effects.so" \
+    -t "${pkgdir}/usr/lib/vst3/${_name} Effects.vst3/Contents/${CARCH}-linux/"
+
+  # install standalone
+  install -vDm 755 "build/surge_xt_products/${_name}" \
+    -t "${pkgdir}/usr/bin/"
+  install -vDm 755 "build/surge_xt_products/${_name} Effects" \
+    -t "${pkgdir}/usr/bin/"
+
+  # install resources
+  install -vdm 755 "${pkgdir}/usr/share/${pkgname}"
+  cp -av resources/data/* "${pkgdir}/usr/share/${pkgname}"
+  install -vDm 644 {AUTHORS,README.md} -t "${pkgdir}/usr/share/doc/${pkgname}"
```

What do you think about removing the `README.md`? IMO it doesn't contain relevant information for users, but maybe we should still keep it?
```suggestion
  install -vDm 644 AUTHORS -t "${pkgdir}/usr/share/doc/${pkgname}"
```

---

seems not useful for users of a binary package, so let's remove it